bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -16,21 +16,31 @@
     (url('foundation.index'), 'index', 'The Mozilla Foundation'),
     (url('foundation.about'), 'about', 'About the Foundation'),
     ('https://mzl.la/foundation-strategy', 'strategy', '2016-2018 Strategy'),
-    (url('foundation.annualreport'), 'report', 'Annual Report'),
-    ('https://careers.mozilla.org/', 'careers', 'Careers'),
-    (donate_url('foundation_nav'), 'donate', 'Donate'),
     (url('foundation.licensing'), 'licensing', 'Licensing & Trademarks'),
     (url('foundation.documents.index'), 'documents', 'Public Documents'),
+    ('https://careers.mozilla.org/', 'careers', 'Careers'),
 ] %}
 
-{% block content %}
-  <main id="main-content">
-    {% block article_title %}{% endblock %}
-
-    <div class="main-column">
-      {% block article %}{% endblock %}
-    </div>
+{% set programs = [
+  (url('foundation.issues'), 'issues', 'Key Internet Issues'),
+  (url('foundation.advocacy'), 'advocacy', 'Advocacy'),
+  (url('foundation.leadership-network'), 'leadership-network', 'Leadership Network')
+] %}
 
+{% block content %}
+  <main>
+    <header class="page-header">
+      <h1><span class="content">{{ _('The Mozilla Foundation') }}</span></h1>
+    </header>
+    <section>
+      <div class="content content-main">
+      <div class="main-column">
+        <header class="section-header">
+        <h1>{% block article_title %}{% endblock %}</h1>
```

Comment:
Correct, that's what I'm suggesting. I can see the appeal of your first approach (only worry about the strings and not the markup) but I think it might come with too many assumptions. Including the markup you actually need along with your content is preferable to embedding markup you won't always need in the parent template and delivering empty elements to the browser that you have to hide with CSS. 

It might also give you more flexibility in the long run. You can put anything you like into that block without being limited to only the elements provided by the parent template.